Extension/src/Debugger/configurationProvider.ts

```diff
@@ -231,11 +231,69 @@ class CppConfigurationProvider implements vscode.DebugConfigurationProvider {
                     config.pipeTransport.pipeProgram = replacedPipeProgram;
                 }
             }
+
+            const macOSMIMode: string = config.osx?.MIMode ?? config.MIMode;
+            const macOSMIDebuggerPath: string = config.osx?.miDebuggerPath ?? config.miDebuggerPath;
+
+            const lldb_10_x_path: string = path.join(util.extensionPath, "debugAdapters", "lldb-mi", "bin", "lldb-mi");
+
+            // Validate LLDB-MI
+            if (os.platform() === 'darwin' && // Check for macOS
+                fs.existsSync(lldb_10_x_path) && // lldb-mi 10.x exists
+                (!macOSMIMode || macOSMIMode === 'lldb') &&
+                !macOSMIDebuggerPath // User did not provide custom lldb-mi
+            ) {
+                const frameworkPath: string = this.getLLDBFrameworkPath();
+
+                if (!frameworkPath) {
+                    const installButton: string = localize("lldb.framework.install.xcode", "Installing XCode");
```

Comment:
@bobbrow This button doesn't install XCode, it takes you to a doc that tells you what the problem is and how the user can solve it. `More Info` or `More Details` would make more sense or something saying `read more` or `How to Install LLDB.Framework`